packages/twenty-front/src/modules/object-metadata/utils/mapObjectMetadataToGraphQLQuery.ts

```diff
@@ -46,22 +46,75 @@ export const mapObjectMetadataToGraphQLQuery = ({
 
   const manyToOneRelationFields = objectMetadataItem?.readableFields
     .filter((field) => field.isActive)
-    .filter((field) => field.type === FieldMetadataType.RELATION)
+    .filter(
+      (field) =>
+        field.type === FieldMetadataType.RELATION ||
+        field.type === FieldMetadataType.MORPH_RELATION,
+    )
     .filter((field) => isDefined(field.settings?.joinColumnName));
 
   const manyToOneRelationGqlFieldWithFieldMetadata =
-    manyToOneRelationFields.map((field) => ({
-      gqlField: field.settings?.joinColumnName,
-      fieldMetadata: field,
+    manyToOneRelationFields.flatMap((field) => {
```

Comment:
ok for now but let's extract this to an util later. I feel we need a objectMetadadata to graphqlFIelds util that can be leverage to generate both:
- recordGqlFields
- gqlQueries